### PR TITLE
Results page

### PR DESCRIPTION
**Pull request by youralien**
New UI for results page

screen shot 2017-02-12 at 2 41 04 pm

Tests:
I entered parameters in the results page, and pressed "Run Results" button, and the proper methods were called successfully on the server.

### CODE REVIEW

File: application/app.py [hunk 1]
```diff
@@ -133,22 +134,28 @@ def test_object(self):
...
-    # for the run button
-    def run(self):
+    # for the runAnalysis button
+    def runAnalysis(self):
         """
         Runs TrafficIntelligence trackers and support scripts.
         """
         #TODO: Make email some internal parameter.
-        remote.analysis(get_config_with_sections(get_config_path(), 'info', 'identifier'))
+        self.api.analysis(get_config_with_sections(get_config_path(), 'info', 'identifier'))
```
Comment (runnersaw) [inline application/app.py]: This is gonna conflict, when you merge make sure this is `api` and not `self.api`.

File: application/app.py [hunk 2]
```diff
@@ -133,22 +134,28 @@ def test_object(self):
...
+
+    def runResults(self):
+        """Runs server methods that generate safety metric results and visualizations"""
+        identifier = get_config_with_sections(get_config_path(), 'info', 'identifier')
+        ttc_threshold = self.ui.timeToCollisionLineEdit.text()
+        vehicle_only = self.ui.vehiclesOnlyCheckBox.isChecked()
+        speed_limit = self.ui.speedLimitLineEdit.text()
+        self.api.results(identifier, ttc_threshold, vehicle_only, speed_limit)
```
Comment (runnersaw) [inline application/app.py]: Make sure this gets changed to `api` when merging dev

**Review by runnersaw — APPROVED** [review]
I didn't test, but this looks great. Glad we're removing trafficintelligence dependencies.

**Review by dcelik — APPROVED** [review]
Looks good! Ship it.

**Pull request merged**